src/main/java/seedu/address/model/appointment/Appointment.java

```diff
@@ -8,7 +8,7 @@
  * Represents an appointment for a patient.
  */
 public abstract class Appointment {
-    public static final String DATE_FORMAT = "dd-MM-yyyy";
+    public static final String DATE_FORMAT = "dd-MM-uuuu";
```

Comment:
I think it may not be a good idea to change `yyyy` to `uuuu` here as this will be the message printed in the result display box and the user will not know what `uuuu` means

Reply:
Fixed!